src/lite-yt-embed.js

```diff
@@ -15,6 +15,7 @@ class LiteYTEmbed extends HTMLElement {
         // Gotta encode the untrusted value
         // https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html#rule-2---attribute-escape-before-inserting-untrusted-data-into-html-common-attributes
         this.videoId = encodeURIComponent(this.getAttribute('videoid'));
+        this.playLabel = this.getAttribute('playlabel') ? encodeURIComponent(this.getAttribute('playlabel')) : 'Play';
```

Comment:
```suggestion
        const playLabel = this.getAttribute('playlabel');
        this.playLabel = playLabel ? encodeURIComponent(playLabel) : 'Play';
```

nothing big changed here. just style and touching the DOM less.

i really like the default label of `Play` and that it matches YT's real one: 

![image](https://user-images.githubusercontent.com/39191/96819563-b70edb80-13d8-11eb-8a54-29b3ece917b1.png)